esp/src/src-react/components/LandingZone.tsx

```diff
@@ -93,33 +82,167 @@ export const LandingZone: React.FunctionComponent<LandingZoneProps> = ({
+    const {
+        selection, setSelection,
+        setTotal,
+        refreshTable } = useFluentStoreState({});
+
+    const [userAddedFiles, setUserAddedFiles] = React.useState<any[]>([]);
+    const [lzPath, setLzPath] = React.useState<string>("");
+    const [pathSep, setPathSep] = React.useState<string>("/");
+
+    const [treeItems, setTreeItems] = React.useState<FlatItem[]>([]);
+    const [openItems, setOpenItems] = React.useState<Iterable<TreeItemValue>>([]);
+
+    const store = useConst(() => userKeyValStore());
+
+    React.useEffect(() => {
+        if (dockpanel) {
+            //  Should only happen once on startup  ---
+            store.get("LzLayout").then(value => {
+                if (!value || value === "undefined") {
+                    const layout: any = dockpanel.layout();
+                    if (Array.isArray(layout?.main?.sizes) && layout.main.sizes.length === 2) {
+                        layout.main.sizes = [0.2, 0.8];
+                        dockpanel.layout(layout).lazyRender();
+                        setLayout(layout);
+                        store?.set("LzLayout", JSON.stringify(layout), true);
+                    }
+                } else {
+                    const layout = JSON.parse(value);
+                    dockpanel.layout(layout);
+                }
+            });
+        }
+    }, [dockpanel, store]);
+
+    React.useEffect(() => {
+        //  Update layout prior to unmount  ---
+        if (dockpanel && store) {
+            return () => {
+                const layout: any = dockpanel.getLayout();
+                store?.set("LzLayout", JSON.stringify(layout), true);
+            };
+        }
+    }, [dockpanel, store]);
 
     React.useEffect(() => {
-        TpDropZoneQuery({}).then(({ TpDropZoneQueryResponse }) => {
-            setTargetDropzones(TpDropZoneQueryResponse?.TpDropZones?.TpDropZone || []);
+        topologyService.TpDropZoneQuery({ ECLWatchVisibleOnly: true }).then(response => {
+            const dropzones = response?.TpDropZones?.TpDropZone || [];
+            if (dropzones[0]?.Path?.indexOf("\\") > -1) {
+                setPathSep("\\");
+            }
+            setDropzones(dropzones);
+            if (dropzones.length) {
+                setSelectedDropzone(dropzones[0]);
+            }
         });
     }, []);
 
+    const addUserFile = React.useCallback((file) => {
+        setData([...data, file]);
+        setUserAddedFiles([...userAddedFiles, file]);
+    }, [data, userAddedFiles]);
+
+    const removeUserFile = React.useCallback((name) => {
+        setData(data.filter(file => file.name !== name));
+        setUserAddedFiles(userAddedFiles.filter(file => file.name !== name));
+    }, [data, userAddedFiles]);
+
+    React.useEffect(() => {
+        if (!dropzones || !selectedDropzone?.Path) return;
+
+        const _path = path ? path.replace(/::/g, "/") : selectedDropzone.Path;
+        setLzPath(_path);
```

Comment:
This isn't the only thing that needs replacing - there are also special chars for upper / lowercase names...